src/dataServices/apiDataService.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    ConditionalDeleteResourceRequest,
+    CreateResourceRequest,
+    DeleteResourceRequest,
+    GenericResponse,
+    PatchResourceRequest,
+    Persistence,
+    ReadResourceRequest,
+    ResourceNotFoundError,
+    UpdateResourceRequest,
+    vReadResourceRequest,
+} from 'fhir-works-on-aws-interface';
+import axios from 'axios';
+
+export class ApiDataService implements Persistence {
+    updateCreateSupported: boolean = false;
+
+    // TODO: Pull this value from AWS Param Store
+    static readonly INTEGRATION_TRANSFORM_URL = 'http://localhost:4000';
```

Comment:
I would have this value passing in from deployment (have deployment deal with param store)

Reply:
This value may not be available during deployment. Our use case: user deploys FHIR Works on AWS and then deploys Integration Transform. This value will be know only after user deploys Integration Transform (including Amazon API Gateway)

Reply:
done